ntbs-service/wwwroot/css/search.scss

```diff
@@ -23,3 +23,91 @@
     margin-left: auto;
     margin-bottom: 0;
 }
+
+.nhs-form-group--search {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.nhs-form-group--search-date {
+    display: inline-block;
+
+    @media(min-width: 450px) {
+        display: flex;
+        flex-wrap: wrap;
+    }
+}
+
+.date-input--search {
+    display: flex;
+    flex-direction: column;
+
+    @media(min-width: 450px) {
+        flex-direction: row;
+    }
+}
+
+.search-label--standard {
+    margin: auto 0;
+    width: 160px;
+}
+
+.search-header {
+    margin-bottom: 12px;
+}
+
+.search-label--long {
+    width: 180px;
+}
+
+.search-input {
+    margin-right: 20px;
+}
+
+.search-date-input {
+    display: flex;
+    margin: 5px 0;
+
+    @media(min-width: 450px) {
+        margin: 0;
+    }
+}
+
+.search-date-input-label {
+    margin: auto 15px auto 0;
+}
+
+.search-dropdown {
+    width: 400px;
+    margin: 15px 0;
+}
+
+.search-sex-radios {
+    display: block;
+    margin: 10px 0 20px 0;
+
+    @media(min-width: 600px) {

```

Comment:
This breakpoint is different to the others (which could maybe be paramaterised?) is there a way to marry these up - then have a single breakpoint section?

Reply:
Yep, I think we can safely have large phone and tablet as the 2 separate breakpoints and add that to a file - @jasiekmiko what are your thoughts?

Reply:
yes, let's use the tablet breakpoint from https://beta.nhs.uk/service-manual/styles-components-patterns/layout as discussed :)